packages/encryption/src/image-steganography/presets.ts

```diff
@@ -108,16 +117,26 @@ const dimensionPreset: readonly Preset[] = [
     },
 ]
 
+export enum SteganographyPreset {
+    Preset2021 = '2021',
+    Preset2022 = '2022',
+    Preset2023 = '2023',
+    Preset202312 = '202312',
```

Comment:
```suggestion
    Preset2023_12 = '2023_12',
```

or Preset2023_Firefly